roofit/roofitcore/CMakeLists.txt

```diff
@@ -460,7 +460,9 @@ ROOT_STANDARD_LIBRARY_PACKAGE(RooFitCore
     RIO
     MathCore
     Foam
-  LINKDEF inc/LinkDef.h
+    Smatrix
```

Comment:
I grepped for Smatrix, and didn't find anything in RooFit. What makes you think that `roofitcore` uses it?

Reply:
@hageboeck Seems to be, you are right. I just checked with code from #5181 - it works also without `Smatrix`. Probably I misinterpret some missing include which were coming from `MathCore`, not from Smatrix. I will extract rootfit changes in separate PR

Reply:
I extract rootfit changes in #5232